migrations/V14__Add_Plan_Payments.sql

```diff
@@ -0,0 +1,147 @@
+CREATE TYPE FUNDING_SOURCE AS ENUM (
```

Comment:
Will need to do a rename after pulling in main now, I think V16 is available

Reply:
Yeah - luckily we now fail to start the app if migrations fails, so this should also get caught in CI (but is good to identify nonetheless!)

Reply:
Perhaps it's a good idea for us to create the migrations as an early mergeable subtask to avoid this specific type of conflict. It doesn't feel great but I've seen this same type of problem many times and that's almost always the fix.